modules/ossm-rn-new-features-1x.adoc

```diff
@@ -19,6 +19,28 @@ Module included in the following assemblies:
 * *Policy Enforcement* - Apply organizational policy to the interaction between services, ensure access policies are enforced and resources are fairly distributed among consumers. Policy changes are made by configuring the mesh, not by changing application code.
 * *Telemetry* - Gain understanding of the dependencies between services and the nature and flow of traffic between them, providing the ability to quickly identify issues.
 
+== New features {SMProductName} 1.1.18.1
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs).
+
+=== Component versions included in {SMProductName} version 1.1.18.1
+
+|===
+|Component |Version
+
+|Istio
+|1.4.8
```

Comment:
It's 1.4.10 for a while, I think we never updated this in the docs.

Reply:
Thanks for pointing this out. The edit has been made.